.gitignore

```diff
@@ -44,3 +44,6 @@ TAGS
 *~
 .DS_Store
 build
+
+.vs
+dev
```

Comment:
Are these created by our build process?

Please add an ending newline to make the diff warning go away.

Reply:
Undid those changes and moved them my local `.git/info/exclude` instead